vms/avm/vm_test.go

```diff
@@ -1473,34 +799,47 @@ func TestForceAcceptImportTx(t *testing.T) {
 		},
 	}
 
+	txAssetID := avax.Asset{ID: avaxID}
 	tx := &txs.Tx{Unsigned: &txs.ImportTx{
 		BaseTx: txs.BaseTx{BaseTx: avax.BaseTx{
 			NetworkID:    constants.UnitTestID,
 			BlockchainID: chainID,
+			Outs: []*avax.TransferableOutput{{
+				Asset: txAssetID,
+				Out: &secp256k1fx.TransferOutput{
+					Amt: 1000,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Threshold: 1,
+						Addrs:     []ids.ShortID{keys[0].PublicKey().Address()},
+					},
+				},
+			}},
 		}},
 		SourceChain: constants.PlatformChainID,
 		ImportedIns: []*avax.TransferableInput{{
 			UTXOID: utxoID,
-			Asset:  avax.Asset{ID: genesisTx.ID()},
+			Asset:  txAssetID,
 			In: &secp256k1fx.TransferInput{
-				Amt:   1000,
-				Input: secp256k1fx.Input{SigIndices: []uint32{0}},
+				Amt: 1010,
```

Comment:
Why change amt for test?

Reply:
This was just to align with the `TestIssueImportTx` function. Technically didn't need to be changed.